test/interp3d_test.cu

```diff
@@ -93,8 +93,8 @@ int main(int argc, char* argv[])
 	if(dplan->opts.gpu_method == 1)
 	{
 		dplan->opts.gpu_binsizex=16;
-		dplan->opts.gpu_binsizey=8;
-		dplan->opts.gpu_binsizez=4;
+		dplan->opts.gpu_binsizey=16;
```

Comment:
Shouldn't we just be using the default bin sizes?

Reply:
I am using the default bin size is 16x16x2. We have to explicitly specify in spread, interp tests because SETUP_BINSIZE is called inside cufinufft_makeplan and is not defined in the header file.

Reply:
Ah I see. Maybe we should add a comment so that people aren't confused.

Reply:
Yes, I agree. I added the comments in 19bdd30.